calculator/Resources/layout/Main.axml

```diff
@@ -4,15 +4,15 @@
     android:layout_width="fill_parent"
     android:layout_height="fill_parent">
     <EditText
-        android:id="@+id/editText2"
+        android:id="@+id/editText1"
```

Comment:
to make the left edit text be the first in the calculation "e.g 4 - 2 is not 2 - 4" it was right to left now it is left to right